examples/flutter_gallery/lib/demo/grid_list_demo.dart

```diff
@@ -310,9 +310,6 @@ class GridListDemoState extends State<GridListDemo> {
     });
   }
 
-  // When the ScrollableGrid first appears we want the last row to only be
-  // partially visible, to help the user recognize that the grid is scrollable.
-  // The GridListDemoGridDelegate's tileHeightFactor is used for this.
```

Comment:
if we still do it, we should just update the comment to be accurate in the new world

if we don't do it, we should probably make sure we do do it again :-)